src/bin/cratesfyi.rs

```diff
@@ -93,7 +93,11 @@ impl CommandLine {
                 Server::start(Some(&socket_addr), reload_templates, config)?;
             }
             Self::Daemon { foreground } => {
```

Comment:
Can you also update the doc-comment on `foreground` to note that it's deprecated?

Reply:
Done.